pkgs/development/python-modules/ifcfg/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, fetchPypi
+, buildPythonApplication
+, iproute2
+}:
+
+buildPythonApplication rec{
+  pname = "ifcfg";
+  version = "0.22";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2960e94e197092877309905dbe18a297c48585f88007be265e0593adae640cd1";
+  };
+
+  propagatedBuildInputs = [
+    iproute2
+  ];
+
+  doCheck = false;
```

Comment:
Please try to run tests as they are available at https://github.com/ftao/python-ifcfg/tree/master/tests

Reply:
I tried running the test, but was unable to get them to run properly.  I couldn't seem to get the `checkPhase` to run in the right directory(?).  In this last push you can see the `checkPhase` I tried.  [Here](https://github.com/ftao/python-ifcfg#development) describes how to run the test.

Reply:
Otherwise please add a comment that currently we'are unable to run tests

Reply:
Sounds good.